package.json

```diff
@@ -44,6 +44,7 @@
     "build:watch:esm": "tsc -p tsconfig.esm.json -w --preserveWatchOutput",
     "clean": "rimraf coverage esm main preload renderer index.* integrations.* ipc.* sentry-electron*.tgz",
     "prelint": "node scripts/update-version.js",
+    "electron-versions": "electron-latest-versions --start 2",
```

Comment:
By looking at the [docs](https://github.com/timfish/electron-latest-versions#cli), major versions are included and may be returned by running this command. I think it's okay to run tests in new patches or even minors, but I don't think we should break CI for tests failing in a new major release. What do you think? Is there something I'm missing?